README.md

```diff
@@ -131,7 +151,18 @@ To uninstall the chart:
       httpGet:
         path: /path
         port: 8080
+      exec: {}
 ```
+### Configuring probes
+
+In order to use probes in deployment, set enable to `true`. After configuring `failureThreshold`, `periodSeconds`, `successThreshold`, `timeoutSeconds`, `initialDelaySeconds`, configure handler type to either `httpGet` or `exec`. You can only use one handler in one probe. `httpGet` is enabled by default, other possible handler is `exec`, whose configuration is specified as below:
```

Comment:
After configuring `failureThreshold`, `periodSeconds`, `successThreshold`, `timeoutSeconds`, `initialDelaySeconds`

I think these are by default defined. so no need to mention it here